contracts/TalentLayerEscrow.sol

```diff
@@ -73,23 +73,23 @@ contract TalentLayerEscrow is Initializable, ERC2771RecipientUpgradeable, UUPSUp
      * @param arbitratorExtraData Extra data to set up the arbitration.
      */
     struct Transaction {
+        Arbitrator arbitrator;
+        bytes arbitratorExtraData;
```

Comment:
This change doesn't optimize gas usage, as the bytes is by default 32 bytes, so can't be packed with the 3 uint16 please revert

Reply:
But the gas reporter shows less gas

Reply:
you can check here the possible option: https://github.com/TalentLayer/talentlayer-id-contracts/pull/132/commits/e455244bb794a1193d795a099272ce6b2119d302

Reply:
it does not answer the question how u count gas optimization, I am pretty sure manual work is less efficient than using gar reporter

Reply:
@0xRomain

Reply:
Yes I used gasReporter too to double check every optim in all commit. Here I focused on the gas used during function execution (not deployment)